rust/noosphere-gateway/src/worker/name_system.rs

```diff
@@ -103,6 +114,53 @@ where
     (tx, task)
 }
 
+/// Run once on gateway start and every PERIODIC_PUBLISH_INTERVAL_SECONDS,
+/// republish all stored link records in gateway spheres that map to
+/// counterpart managed spheres.
+async fn periodic_publisher_task<C, K, S>(
+    tx: UnboundedSender<NameSystemJob<C>>,
+    local_spheres: Vec<C>,
+) where
+    C: HasMutableSphereContext<K, S>,
+    K: KeyMaterial + Clone + 'static,
+    S: Storage + 'static,
+{
+    loop {
+        for local_sphere in local_spheres.iter() {
+            if let Err(error) = periodic_publish_record(&tx, local_sphere).await {
+                error!("Could not publish record: {}", error);
+            };
+        }
+        tokio::time::sleep(Duration::from_secs(PERIODIC_PUBLISH_INTERVAL_SECONDS)).await;
```

Comment:
Can you also pull the time for the other periodic task out into its own const so that we can see them together at a glance?